docs/source/markdown/podman-run.1.md

```diff
@@ -688,6 +688,11 @@ Note that the container will not be removed when it could not be created or
 started successfully. This allows the user to inspect the container after
 failure.
 
+**--rmi**=*true|false*
+
+Automatically remove the container image when it exits
```

Comment:
After exit of the container, remove the image unless another ....

Reply:
suggest:  "After the container exits, remove the image unless another ..."